pkg/clustermanager/cluster_manager.go

```diff
@@ -1176,6 +1178,16 @@ func (c *ClusterManager) ApplyBundles(ctx context.Context, clusterSpec *cluster.
 	if err != nil {
 		return fmt.Errorf("applying bundle spec: %v", err)
 	}
+
+	// We need to update this config map with the new upgrader images whenever we
+	// apply a new Bundles object to the cluster in order to support in-place upgrades.
+	cm, err := getUpgraderImagesFromBundle(clusterSpec)
+	if err != nil {
+		return fmt.Errorf("getting upgrader images from bundle: %v", err)
+	}
+	if err = c.clusterClient.Apply(ctx, cluster.KubeconfigFile, cm); err != nil {
```

Comment:
Does this apply replace the configmap with new contents or does it merge with the current configmap if present on the cluster?

If it's doing a replace, we might have to fetch the cm in the code and append to it ourselves (if it exists) and then apply it back.

Reply:
Yeah we did decide to append it. Updated this to fetch the existing one